File: backend/src/app/models/ApAvailableOid.js
import Sequelize, { Model } from 'sequelize';

class ApAvailableOid extends Model {
    static init(sequelize) {
        super.init(
            {
                description: Sequelize.STRING,
                oid: Sequelize.STRING,
                oid_reduced: Sequelize.STRING,
                ap: Sequelize.STRING,

            },
            { tableName: 'ap_available_oid', sequelize, timestamps: false }
        );

        return this;
    }

    static associate(models) {
        this.belongsTo(models.Ap, { foreignKey: 'ap_id', as: 'ap' });
    }
}

export default ApAvailableOid;

File: snmpagent/app/models/ApAvailableOid.py
from django.db import models
from app.models.Ap import Ap

class ApAvailableOid(models.Model):
    description = models.CharField(max_length=100)
    oid = models.CharField(max_length=60, null=True, blank=True)
    oid_reduced = models.CharField(max_length=60, null=True, blank=True)
    ap = models.ForeignKey(Ap, on_delete=models.CASCADE, null=True, default=None, blank=True)

    class Meta:
        db_table = "ap_available_oid"